pkg/controller/machineset/controller.go

```diff
@@ -333,10 +333,10 @@ func (c *ReconcileMachineSet) waitForMachineCreation(machineList []*clusterv1alp
 			err := c.Client.Get(context.Background(),
 				client.ObjectKey{Namespace: machine.Namespace, Name: machine.Name},
 				&clusterv1alpha1.Machine{})
-			klog.Error(err)
 			if err == nil {
 				return true, nil
 			}
+			klog.Error(err)
```

Comment:
/lgtm

Reply:
(Though we really should deprecate this method in klog, and force an error message!)